### PR TITLE
docs: add link to Yarn in README

### PR DESCRIPTION
**Pull request by Ana06**
Remove the code markdown. It is not code, it is just a name. :bowtie: 

## PR Checklist

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: aio/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Everything in this folder is part of the documentation project. This includes
 
 ## Developer tasks
 
-We use `yarn` to manage the dependencies and to run build tasks.
+We use [Yarn](https://yarnpkg.com/lang/en) to manage the dependencies and to run build tasks.
```
Comment (gkalpak) [inline aio/README.md]: Can you please change the link to just `https://yarnpkg.com/`

**Review by gkalpak — CHANGES_REQUESTED** [review]

**Comment by gkalpak** [comment]
Thx for the PR! Can you also change the commit message to `docs(docs-infra): ...` (we've recently switched from `aio` to `docs-infra`, which is more descriptive).

**Comment by Ana06** [comment]
@gkalpak is there a way to mark the commit as trivial so that there is not need to sign the CLA? If not, I would probably need to ask my company, as I did this in work time...

**Comment by brandonroberts** [comment]
@Ana06 signing the CLA is required to contribute: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-signing-the-cla

**Comment by Ana06** [comment]
A lot of work to update a link... :sweat: But I think it is now ready! :wink:

**Review by brandonroberts — APPROVED** [review]

**Review by gkalpak — APPROVED** [review]

**Comment by brandonroberts** [comment]
Thanks @Ana06!

**Comment by benlesh** [comment]
Merged. Congrats, @Ana06, and thank you.

**Pull request closed without merging**